src/main/java/core/basesyntax/operations/PurchaseOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.dto.ProductDto;
+import core.basesyntax.storage.Storage;
+
+public class PurchaseOperation implements Operation {
+    @Override
+    public void apply(ProductDto productDto) {
+        if (productDto.getQuantity() < 0
```

Comment:
If quantity less than 0, we have to throw Exception

![image](https://user-images.githubusercontent.com/80862260/117122475-9ddee280-ad9e-11eb-9fb4-411405fe65ce.png)